py-polars/polars/dataframe/frame.py

```diff
@@ -3120,7 +3120,7 @@ def write_excel(
             xlv = xlsxwriter.__version__
             if parse_version(xlv) < parse_version("3.0.8"):
                 raise ModuleNotFoundError(
-                    f"`autofit=True` requires xlsxwriter 3.0.8 or higher, found {xlv}"
+                    f"`autofit=True` requires xlsxwriter 3.0.8 or higher, found {xlv!r}"
```

Comment:
This will lead to quotes around the version number, which I think doesn't make for nice formatting.